compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/declarations/IrDeclarationOrigin.kt

```diff
@@ -32,6 +32,7 @@ interface IrDeclarationOrigin {
     object FILE_CLASS : IrDeclarationOriginImpl("FILE_CLASS")
     object GENERATED_DATA_CLASS_MEMBER : IrDeclarationOriginImpl("GENERATED_DATA_CLASS_MEMBER")
     object GENERATED_INLINE_CLASS_MEMBER : IrDeclarationOriginImpl("GENERATED_INLINE_CLASS_MEMBER")
+    object SYNTHETIC_INLINE_CLASS_MEMBER : IrDeclarationOriginImpl("SYNTHETIC_INLINE_CLASS_MEMBER", isSynthetic = true)
```

Comment:
Let's add this to `JvmLoweredDeclarationOrigin` instead. These origins are used in IR serialization format in Native and JS (`KotlinIr.proto`) and it's probably better to avoid adding JVM specifics here at this point

Reply:
Thanks for the explanation, I wasn't aware of this! Done.